runtime/compiler/optimizer/J9RecognizedCallTransformer.cpp

```diff
@@ -367,12 +367,16 @@ bool J9::RecognizedCallTransformer::isInlineable(TR::TreeTop* treetop)
       case TR::java_lang_Class_isAssignableFrom:
          return cg()->supportsInliningOfIsAssignableFrom();
       case TR::java_lang_Integer_rotateLeft:
+      case TR::java_lang_Integer_rotateRight:
+         return TR::Compiler->target.cpu.isX86() || TR::Compiler->target.cpu.isZ() || TR::Compiler->target.cpu.isPower();
       case TR::java_lang_Long_rotateLeft:
+      case TR::java_lang_Long_rotateRight:
+         return TR::Compiler->target.cpu.isX86() || TR::Compiler->target.cpu.isZ() || (TR::Compiler->target.cpu.isPower() && TR::Compiler->target.is64Bit());
       case TR::java_lang_Math_abs_I:
       case TR::java_lang_Math_abs_L:
       case TR::java_lang_Math_abs_F:
       case TR::java_lang_Math_abs_D:
-         return TR::Compiler->target.cpu.isX86() || TR::Compiler->target.cpu.isZ();
+         return true;
```

Comment:
Just double checking: it's ok to return true here? E.g. all platforms support abbs opcodes?

Reply:
As far as I can tell, all codegens (including ARM and AArch64) do currently support `iabs`, `labs`, `fabs`, and `dabs`.

Reply:
I would prefer enabling this for ARM and AArch64 to happen at the very least in a separate commit.  It is enabling new functionality for those platforms and should be isolated.  It also doesn't fit the theme of this PR.

Reply:
Fair point. I've amended this PR to only enable this transformation on Power and it can be enabled on ARM/AArch64 as part of a separate PR at a later date.

Reply:
Thanks.  Issue #7162 to track this.